README.md

```diff
@@ -2,7 +2,11 @@
 
 Rebuild your Jekyll site when a file changes with the `--watch` switch.
 
-[![Build Status](https://travis-ci.org/jekyll/jekyll-watch.svg?branch=master)](https://travis-ci.org/jekyll/jekyll-watch)
+[![Linux Build Status](https://travis-ci.org/jekyll/jekyll-watch.svg?branch=master)][travis]
```

Comment:
I'd go with shields.io here too because it'll be faster and consistent. Also, I'd add `/master` here too.